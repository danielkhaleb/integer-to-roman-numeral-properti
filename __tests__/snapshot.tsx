import { render } from '@testing-library/react'
import CalculatorIntegerToRomanNumeral from '@/pages/index'

it('renders homepage unchanged', () => {
  const { container } = render(<CalculatorIntegerToRomanNumeral />)
  expect(container).toMatchSnapshot()
})
