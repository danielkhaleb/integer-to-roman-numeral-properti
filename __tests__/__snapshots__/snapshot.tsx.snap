// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="sc-gsnTZi bXZAsw"
  >
    <section
      class="sc-dkzDqf etqhMt"
    >
      <h1>
        Convert number to roman number
      </h1>
      <section
        class="sc-bczRLJ bMbtBZ"
      >
        <input
          data-testid="inputComponent"
          placeholder="Please insert a integer number"
          type="number"
          value=""
        />
      </section>
    </section>
  </main>
</div>
`;
